src/main/java/org/codehaus/mojo/buildhelper/AddResourceMojo.java

```diff
@@ -37,10 +38,22 @@
 @Mojo(name = "add-resource", defaultPhase = LifecyclePhase.GENERATE_RESOURCES, threadSafe = true)
 public class AddResourceMojo extends AbstractAddResourceMojo {
 
+    /**
+     * Skip plugin execution.
+     *
+     * @since 3.5.0
+     */
+    @Parameter(property = "buildhelper.addresource.skip", defaultValue = "false")
+    private boolean skip;
```

Comment:
Greate we have a possibility to skip each goal by separate property, we should have such possibility by configuration item, so I please rename also field according to property name.

As we have `skipAttach` with property `buildhelper.skipAttach`, I would like:
`skipAddresource` with property `buildhelper.skipAddresource` to preserve existing convention.

Reply:
Ok, I see in #187 we will have `buildhelper.addresource.skipIfMissing` ... 
so `buildhelper.addresource.skip` looks ok  😄